src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-falcon.py

```diff
@@ -0,0 +1,27 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2024 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+hiddenimports = [
+    'cgi'
```

Comment:
There's a comma missing here at the end, which causes the the first hidden import to become `cgifalcon.app_helpers`, and hence we miss both `cgi` and `falcon.app_helpers`.

Reply:
Well spotted! I know exactly what happened: I used the VSCode 'Sort' function and TIL it's not smart enough to fix the commas 🙄